packages/next/src/build/index.ts

```diff
@@ -1795,6 +1798,65 @@ export default async function build(
         if (!nodeFileTrace) {
           nodeFileTrace =
             require('next/dist/compiled/@vercel/nft').nodeFileTrace
+          nftResolve = require('next/dist/compiled/@vercel/nft').resolve
+          resolveHook = async (
+            id: string,
+            parent: string,
+            job: any,
+            isCjs: boolean
+          ) => {
+            if (id[0] !== '.' && id[0] !== '/' && id[0] !== '\\') {
```

Comment:
small Q: when the `id[0]` will be the slash?

Reply:
rare but if you were requiring from the root like `require('/users/myuser/myproject/somepackage')` practically speaking this would only happen if you used `path.resolve` or something to produce a specifier